ffi/derive/tests/ui_fail/derive_skip_field.rs

```diff
@@ -4,9 +4,9 @@ use getset::{Getters, Setters};
 use iroha_ffi::{ffi_export, FfiConvert, FfiType};
 
 /// FfiStruct
+#[ffi_export]
```

Comment:
has to be outer most macro so that it's expanded before `derive` macro

Reply:
I would normally prefer a function like macro when macro's expansion is position dependent but in this case this behavior seemed appropriate. I don't see how `no_std` and `no_main` relate to this. I've added a description